### PR TITLE
TELCODOCS#1897: 4.16 LVMS Release Notes

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.16

Issue:
[TELCODOCS-1897](https://issues.redhat.com/browse/TELCODOCS-1897)

Link to docs preview:
- [Storage](https://77347--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-storage_release-notes)
- [Bug fixes](https://77347--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-16-release-notes.html#ocp-4-16-bug-fixes_release-notes)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: release_notes/ocp-4-16-release-notes.adoc [hunk 1]
```diff
@@ -189,6 +189,36 @@ For the full list of available secrets store providers, see xref:../nodes/pods/n
...
 For information about using the {secrets-store-operator} to mount secrets from HashiCorp Vault, see xref:../nodes/pods/nodes-pods-secrets-store.adoc#secrets-store-vault_nodes-pods-secrets-store[Mounting secrets from HashiCorp Vault].
 
+[id="ocp-4-16-storage-thick-provisioning_{context}"]
+==== Support for configuring thick-provisioned storage
+
+This feature provides support for configuring thick-provisioned storage. If you exclude the `deviceClasses.thinPoolConfig` field in the `LVMCluster` custom resource (CR), logical volumes are thick provisioned.
+Using thick-provisioned storage includes the following limitations:
+
+* No copy-on-write support for volume cloning.
+* No support for snapshot class.
```
[the remark this answers is not in the document]
Comment (sr1kar99) [reply release_notes/ocp-4-16-release-notes.adoc:199]: Updated as follows:

"No support for `VolumeSnapshotClass`. Therefore, CSI snapshotting is not supported."

**Comment by sr1kar99** [comment]
Hi @jakobmoellerdev 
Updated the content as per the suggestions.
Could you PTAL?
Thanks!

**Review by jakobmoellerdev — COMMENTED** [review]
/lgtm

**Comment by sr1kar99** [comment]
Hi @radeore 
Could you please review this PR?
Thanks!

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Review by sheriff-rh — APPROVED** [review]
Changes LGTM, merging.

**Pull request merged**